### PR TITLE
Demo UI feature for Manual Filters and Aggregations on input query

### PR DESCRIPTION
**Pull request by sohamkasar19**
1. Updates from BB demo UI
2. New Feature for Manual Filters and Aggregations on Input Query
- Can be turned on using the "auto-filter" flag

### CODE REVIEW

File: apps/demo-ui/ui/src/Types.tsx [hunk 1]
```diff
@@ -53,7 +53,8 @@ export class SystemChat {
   fromAdhoc: boolean = false;
   editing: boolean = false;
   comment: string = "";
-  filterContent: any;
+  filterContent: FilterValues = {};
```
Comment (baitsguy) [inline apps/demo-ui/ui/src/Types.tsx:56]: can you make the names consistent: filtersUser or aggregationValues

File: apps/demo-ui/ui/src/pages/HomePage/components/Chatbox/AddAggregationModal.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import { Dispatch, SetStateAction, useEffect, useState } from "react";
+import { documentLevelFields } from "../../../../utils/ChatboxUtils";
+import { AggregationValues } from "../../../../Types";
+
+export const AddAggregationModal = ({
+  addAggregationsModalOpened,
+  addAggregationsModalhandlers,
+  aggregations,
+  setAggregations,
+  filterFields,
```
Comment (baitsguy) [inline apps/demo-ui/ui/src/pages/HomePage/components/Chatbox/AddAggregationModal.tsx:18]: aggregationFields?

File: apps/demo-ui/ui/src/pages/HomePage/components/Chatbox/SystemChatbox.tsx [hunk 3]
```diff
@@ -80,10 +90,17 @@ export const SystemChatBox = ({
...
 }) => {
   const citationRegex = /\[(\d+)\]/g;
   const theme = useMantineTheme();
-  console.log("Filter content is", systemChat.filterContent);
+  const [openSearchResultOpened, openSearchResultHandler] =
+    useDisclosure(false);
+  const [openSearchQueryModalOpened, openSearchQueryModalHandler] =
+    useDisclosure(false);
+
+  // console.log("Filter content is", systemChat.filterContent);
```
Comment (baitsguy) [inline apps/demo-ui/ui/src/pages/HomePage/components/Chatbox/SystemChatbox.tsx:103]: delete

**Review by baitsguy — APPROVED** [review]

File: apps/demo-ui/ui/src/Types.tsx (hunk 1 above)
Comment (sohamkasar19) [reply apps/demo-ui/ui/src/Types.tsx:56]: Yes

File: apps/demo-ui/ui/src/pages/HomePage/components/Chatbox/SystemChatbox.tsx (hunk 3 above)
Comment (sohamkasar19) [reply apps/demo-ui/ui/src/pages/HomePage/components/Chatbox/SystemChatbox.tsx]: Done

File: apps/demo-ui/ui/src/pages/HomePage/components/Chatbox/AddAggregationModal.tsx (hunk 2 above)
Comment (sohamkasar19) [reply apps/demo-ui/ui/src/pages/HomePage/components/Chatbox/AddAggregationModal.tsx]: Yes, updated

**Pull request merged**